docs/README.md

```diff
@@ -50,6 +50,42 @@ Prerequisite: In the API (packages/api) exists a corresponding GraphQL endpoint.
 4) Now the file `index.ts` will be generated automatically `in editor/src/client/api`
 5) Now you can import your desired endpoint in your .tsx file. See for example `subscriptionEditPanel.tsx`
 
+
+### Analytics
+In order to use analytics and send data to WePublish's Matomo, we prepared an analytics.js plugin that's available here:
```

Comment:
I would not really mention about matomo directly here. Only note on the very bottom.